### DISCUSSION TITLE
accidently deleted def

### DISCUSSION THREAD

**Discussion by iexist12** [Monika-After-Story/MonikaModDev#9038] [QA - tech support + general] [answered]
theres supposed to be all of monikas sprites in the def file i think but its not there. its supposed to go ddlc/game/mod_assets/monika/c/def but i accidentally deleted def. not sure what to do now.

**Comment by CrocandCo** [comment]
I can send you a copy of my file if you want, or re-download the mod manually.

**Comment by multimokia** [comment, marked as answer]
Simply manually reinstall the mod.
